frontend/src/components/layout/Navbar.vue

```diff
@@ -32,6 +33,16 @@ const { getKeycloak } = storeToRefs(useAuthStore());
         </ol>
       </template>
     </Toolbar>
+
+    <!-- Breadcrumb
```

Comment:
Wondering if we need to keep the commented code?

Reply:
For this one, we want to revisit breadcrumbs at a later date, so this should be preserved.